docs/getting-started/installation.md

```diff
@@ -3,6 +3,20 @@ id: installation
 title: Installation
 ---
 
+Rest Hooks is a library for fetching structured data in performant way with no boilerplate.
+
+Its interface is declarative and minimal. A small structure declaration is all that is needed
+for Rest Hooks to perform numerous fetching and caching optimizations while providing predictable,
+precisely typed data to consume.
+
+Structured data means data composed of [Objects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
+([maps](https://en.wikipedia.org/wiki/Associative_array))
+and [Arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array)
+([lists](https://en.wikipedia.org/wiki/List_(abstract_data_type))), as opposed media
+like images and videos. This makes it great for API calls regardless of form ([REST-like](https://restfulapi.net/),
+[GraphQL](https://graphql.org/), [gRPC](https://grpc.io/)), serialization ([JSON](llhttps://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON), [YAML](https://en.wikipedia.org/wiki/YAML)),
+or transport ([HTTP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview), [WebSockets](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API), [local](../guides/mocking-unfinished)).
+
```

Comment:
would be cool if you added a section comparing it to alternatives

Reply:
mmmm, i have consciously tried to avoid mentioning any alternatives. what would be the benefit?

PS) my primary audience is people using redux with REST apis so I try to frame everything in that way

Reply:
Not to say I don't think comparisons are useful - just that I don't believe they belong in an introduction to this library. A standalone blog post I feel is the best place for such things as they tend to be temporally limited.